build/shade/_dotnet-build.shade

```diff
@@ -12,10 +12,17 @@ configuration=''
 
 default configuration = 'Debug'
 default build_options=' ${E("KOREBUILD_DOTNET_BUILD_OPTIONS")}'
+default framework = null
 
 @{
     var projectFolder=Path.GetDirectoryName(projectFile);
 
     var dotnetArgs=string.Format("build{0} {1} --configuration {2}", build_options, projectFolder, configuration);
+
+    if (!string.IsNullOrEmpty(framework))
+    {
+        dotnetArgs=string.Format("build{0} {1} --configuration {2} --framework {3}", build_options, projectFolder, configuration, framework);
```

Comment:
Isn't there an environment variable to pass in options already? Why not use that?